src/main/java/com/jpaplayground/conversation/UserType.java

```diff
@@ -0,0 +1,15 @@
+package com.jpaplayground.conversation;
+
+public enum UserType {
+	Seller,
+	Buyer,
+	All;
+
+	public UserType counterPart() {
+		if (this.equals(Seller)) {
+			return Buyer;
+		} else {
+			return Seller;
+		}
```

Comment:
```suggestion
		return Seller;
```

`else` 블록은 제외시켜도 될 듯?